packages/marko-web-identity-x/browser/profile.vue

```diff
@@ -234,6 +238,18 @@ export default {
       type: Array,
       default: () => [],
     },
+    eventName: {
+      type: String,
+      default: 'profile',
+    },
```

Comment:
`eventName` isn't used within this component, only `viewEventName` and `submitEventName`.

Reply:
I removed the eventName prop that was never used instead handle with the viewEventName & submitEventName props